pyTST/core.py

```diff
@@ -204,8 +228,12 @@ def update_cursor(index):
 
                 hline.set_ydata(min_u95)
                 vline.set_xdata(self.step_time_array[index])
+                ax0.loglog(self.step_time_array[:self.step_time_array.size - (self.step_time_array.size-index)], self.u95_array[:(self.step_time_array.size-index-1):-1], color='C0')
+                ax0.loglog(self.step_time_array[self.step_time_array.size - (self.step_time_array.size-index):], self.u95_array[(self.step_time_array.size-index-1)::-1], color='C1')
```

Comment:
these lines are too long and should have at least one line break.
also self.step_time_array.size is used a lot, storing it in a variable will make it easier to read